packages/core2/src/constants/project.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Slice Machine's published NPM package name.
+ */
+export const SLICE_MACHINE_NPM_PACKAGE_NAME = "slice-machine-ui";
+
+/**
+ * Slice Machine configuration filename.
+ */
```

Comment:
This TSDoc would only apply to the first constant. If we want to comment in the file to group code, we should use `//` comments instead.